Re-write the `Silence` wiki page (#12505)
ppy/osu-wiki: wiki/Silence/en.md

# Silence

> **Silence** is a term used to describe the punishment of limiting a user's means of communications within the osu! community. Members of the [Global Moderation Team](/wiki/People/Global_Moderation_Team) and [Nomination Assessment Team](/wiki/People/Nomination_Assessment_Team) typically issue these silences to maintain a clean environment both on the website and in-game.

Reviewer: Add comma since "[...]maintain a clean environment." can end the sentence, "both on the website and in-game." is an extension.
```suggestion
**Silence** is the term used to describe the punishment of limiting a user's means of communications within the osu! community. Members of the [Global Moderation Team](/wiki/People/Global_Moderation_Team) and [Nomination Assessment Team](/wiki/People/Nomination_Assessment_Team) typically issue these silences to maintain a clean environment, both on the website and in-game.
```

> 

Reviewer: Add comma since "[...]maintain a clean environment." can end the sentence, "both on the website and in-game." is an extension.
```suggestion
**Silence** is a term used to describe the punishment of limiting a user's means of communications within the osu! community. Members of the [Global Moderation Team](/wiki/People/Global_Moderation_Team) and [Nomination Assessment Team](/wiki/People/Nomination_Assessment_Team) typically issue these silences to maintain a clean environment, both on the website and in-game.
```

## Limitations

::: Infobox
![](img/silence-profile.png "Silence record in the profile page")
:::

A user who has an active silence cannot do the following:

- Use the online [chat](/wiki/Client/Interface/Chat_console) (public and private), both in-game and on the website.
- Post on any of the osu! forums.
- Post comments anywhere on the website (beatmaps, changelog, news).
- Take part in [beatmap discussions](/wiki/Beatmap_discussion).
- Edit profile details, change the avatar, cover and userpage.
- Submit and update beatmaps.
- Partake in [multiplayer](/wiki/Client/Interface/Multiplayer) games.

::: Infobox
![](img/silence-ingame.png "Silence warning in the chat")
:::

Once a silence is issued to a user, all of their messages in chat are removed and an alert will be displayed in the place of the chat input field containing the duration of the active silence.[^chat-cleanup]

## Durations

> The duration of a silence starts at 5 minutes and doubles for each subsequent offence, up to a maximum of 28 days, but the initial duration may vary depending on the offence committed and the user's prior history. The factors considered when determining the length of a silence include:

Reviewer: "generally" for consistency with #12441
```suggestion
The duration of a silence starts at 5 minutes and generally doubles for each subsequent offence, up to a maximum of 28 days, but the initial duration may vary depending on the offence committed and the user's prior history. The factors considered when determining the length of a silence include:
```

> 

Reviewer: "generally" from PR #12441
```suggestion
The duration of a silence starts at 5 minutes and generally doubles for each subsequent offence, up to a maximum of 28 days, but the initial duration may vary depending on the offence committed and the user's prior history. The factors considered when determining the length of a silence include:
```

- **Atmosphere:** The current chat environment's state.
- **Frequency:** Repetitiveness regardless of previous silences given.
- **History:** Records of previous infringements.
- **Severity:** The seriousness of the offence committed.

> In some cases, users may receive a final warning before an account restriction is placed, providing an opportunity to cease the offending behaviour.

Reviewer: ```suggestion
In some cases, users may receive a final warning before an [account restriction](/wiki/Help_centre/Account_restrictions) is placed, providing an opportunity to cease the offending behaviour.
```

> 

Reviewer: ```suggestion
In some cases, users may receive a final warning before an [account restriction](/wiki/Help_centre/Account_restrictions) is placed, providing an opportunity to cease the offending behaviour.
```

## Common reasons for silences

> Here are some common reasons why users get silenced in public chat, but not limited to:

Reviewer: ```suggestion
Some common reasons why users get silenced in public chat include, but are not limited to:
```

> 

Reviewer: ```suggestion
Some common reasons why users get silenced in public chat include, but are not limited to:
```

- **Spamming or flooding:** Self-explanatory.
- **Caps abuse:** Chatting with all uppercase letters.

> - **Inappropriate behavior/conversation:** The public chat is not a place for such discussions.

Reviewer 2: ```suggestion
- **Inappropriate behaviour/conversation:** The public chat is not a place for such discussions.
```

Reviewer: IMO there should be a brief description of what discussions are being referred to here.
I would think something like this:
```suggestion
- **Inappropriate behavior/conversation:** The public chat is not a place for discussions not appropriate for all ages, or for heavily inflammatory topics.
```

Author: i would think any inflammatory topic is not welcome so maybe the `heavily` is not needed

Author: @chromb thoughts on this? asking cuz of https://github.com/ppy/osu-wiki/pull/12505#discussion_r1866769530

> - **Racism:** Discrimination or hatred based on race, religion, gender, or sexual orientation.

Reviewer: ```suggestion
- **Racism:** Discrimination or hatred based on race, religion, gender, sexual orientation, etc.
```

> - **Advertising:** Promoting products or services. This includes Discord invite links and streaming links such as Twitch and YouTube.

Reviewer: "Such discussions" is entirely undescriptive.
```suggestion
- **Inappropriate behavior/conversation:** The public chat is not a place for discussions not appropriate for all ages, or for heavily inflammatory topics.
```

Reviewer 3: I would argue that this should just be a 'General bad behaviour and/or inappropriate conversation topics.'

> - **Advertising:** Promoting products or services. This includes Discord invite links and streaming links such as Twitch and YouTube.

Reviewer: ```suggestion
- **Racism:** Discrimination or hatred based on race, religion, gender, sexual orientation, etc.
```

- **Unwanted content:** Websites containing piracy, screamers, referrals and the like. 

Silences may also be given to users for other reasons such as but not limited to:

> 

Reviewer: ```suggestion
Silences may also be given to users for other reasons such as, but not limited to:
```

> Silences may also be given to users for other reasons such as but not limited to:
> 
> - **Inappropriate beatmap submission:** Details such as metadata, tags and background image.

Reviewer: Needs a serial comma (and inappropriate difficulty names are not that uncommon).
```suggestion
Silences may also be given to users for other reasons, such as (but not limited to):

- **Inappropriate beatmap submission:** Details such as metadata, tags, difficulty names, and background images.
```

> - **Inappropriate conduct:** Places such as the forums, beatmap discussions and comments.

Reviewer: Serial comma (and inappropriate difficulty names are not very uncommon)
```suggestion
- **Inappropriate beatmap submission:** Details such as metadata, tags, difficulty names, and background images.
```

# Silence

> **Silence** is a term used to describe the punishment of limiting a user's means of communications within the osu! community. Members of the [Global Moderation Team](/wiki/People/Global_Moderation_Team) and [Nomination Assessment Team](/wiki/People/Nomination_Assessment_Team) typically issue these silences to maintain a clean environment for the website and in-game.

Reviewer 4: sounds like there should be something after "in-game" otherwise

```suggestion
**Silence** is a term used to describe the punishment of limiting a user's means of communications within the osu! community. Members of the [Global Moderation Team](/wiki/People/Global_Moderation_Team) and [Nomination Assessment Team](/wiki/People/Nomination_Assessment_Team) typically issue these silences to maintain a clean environment both on the website and in-game.
```

## Limitations

::: Infobox
![](img/silence-profile.png "Silence record in the profile page")
:::

A user who has an active silence cannot do the following:

> - Use the online [chat](/wiki/Client/Interface/Chat_console) (public and private), both in-game and on the website
> - Post on any of the osu! forums

Reviewer 4: like the other lists in this pr

```suggestion
- Use the online [chat](/wiki/Client/Interface/Chat_console) (public and private), both in-game and on the website.
- Post on any of the osu! forums.
```

> - Post comments anywhere on the website (beatmaps, changelog, news)
> - Take part in [beatmap discussions](/wiki/Beatmap_discussion)

Reviewer 4: ```suggestion
- Post comments anywhere on the website (beatmaps, changelog, news).
- Take part in [beatmap discussions](/wiki/Beatmap_discussion).
```

> - Edit profile details such as avatar and cover
> - Submit and update beatmaps
> - Partake in [multiplayer](/wiki/Client/Interface/Multiplayer) games

Reviewer 4: ```suggestion
- Edit profile details such as the avatar and cover.
- Submit and update beatmaps.
- Partake in [multiplayer](/wiki/Client/Interface/Multiplayer) games.
```

Once a silence is issued to a user, all of their messages in chat are removed and an alert will be displayed in the place of the chat input field containing the duration of the active silence.

::: Infobox
![](img/silence-ingame.png "Silence warning in the chat")
:::

## Durations

> The duration of a silence starts at 5 minutes and doubles for each subsequent offence, up to a maximum of 28 days, but the initial duration may vary depending on the offence committed and the user's previous history. The factors considered when determining the length of a silence include:

Reviewer 4: ```suggestion
The duration of a silence starts at 5 minutes and doubles for each subsequent offence, up to a maximum of 28 days, but the initial duration may vary depending on the offence committed and the user's prior history. The factors considered when determining the length of a silence include:
```

- **Atmosphere:** The current chat environment's state.

> - **Frequency:** Repetitiveness regardless of the previous silences given.

Reviewer 4: ```suggestion
- **Frequency:** Repetitiveness regardless of previous silences given.
```

> - **History:** The previous records of infringements.

Reviewer 4: ```suggestion
- **History:** Records of previous infringements.
```

- **Severity:** The seriousness of the offence committed.

In some cases, users may receive a final warning before an account restriction is placed, providing an opportunity to cease the offending behaviour.

## Common reasons for silences

Here are some common reasons why users get silenced in public chat, but not limited to:

> - **Spamming or Flooding:** Self-explanatory.
> - **Caps Abuse:** Chatting with all uppercase letters.
> - **Inappropriate Behavior/Conversation:** The public chat is not a place for such discussions.
> - **Racism:** Discrimination or hatred based on race, religion, gender or sexual orientation.
> - **Advertising:** Promoting products or services. This includes Discord invite links and streaming links such as Twitch and YouTube.
> - **Unwanted Content:** Websites such as piracy, screamers and referrals. 

Reviewer 4: we don't typically use title case in lists like these

```suggestion
- **Spamming or flooding:** Self-explanatory.
- **Caps abuse:** Chatting with all uppercase letters.
- **Inappropriate behavior/conversation:** The public chat is not a place for such discussions.
- **Racism:** Discrimination or hatred based on race, religion, gender, or sexual orientation.
- **Advertising:** Promoting products or services. This includes Discord invite links and streaming links such as Twitch and YouTube.
- **Unwanted content:** Websites containing piracy, screamers, referrals and the like. 
```

Silences may also be given to users for other reasons such as but not limited to:

> - **Inappropriate Beatmap Submission:** Details such as metadata, tags and background image.
> - **Inappropriate Conduct:** Places such as the forums, beatmap discussions and comments.
> - **Inappropriate Profile:** Contents such as avatar, cover and profile details.

Reviewer 4: ```suggestion
- **Inappropriate beatmap submission:** Details such as metadata, tags and background image.
- **Inappropriate conduct:** Places such as the forums, beatmap discussions and comments.
- **Inappropriate profile:** Content such as avatars, covers and profile details.
```

## Appealing

> While silences are implemented to maintain a positive community environment, it's acknowledged that errors can occur. If you believe that the silence issued to you was an error or unjustified, kindly contact the [account support team](/wiki/People/Account_support_team#accounts@ppy.sh) at [accounts@ppy.sh](mailto:accounts@ppy.sh) and explain the situation.

Reviewer 4: ```suggestion
While silences are implemented to maintain a positive community environment, errors can occur. If you believe that the silence issued to you was an error or unjustified, kindly contact the [account support team](/wiki/People/Account_support_team#accounts@ppy.sh) at [accounts@ppy.sh](mailto:accounts@ppy.sh) and explain the situation.
```
